src/test/resources/features/Returns.feature

```diff
@@ -121,47 +122,47 @@ Feature: Submit ECL Return
     When I enter 12 month accounting period revenue <UK Revenue> and select Yes for my AML-regulated activity for the full financial year
     Then I should be on the page that says <Expected content>
     Examples:
-      | UK Revenue | Expected content                                                        |
-      | 10200000   | The amount of levy you need to pay for this financial year is £10,000.  |
-      | 35999999   | The amount of levy you need to pay for this financial year is £10,000.  |
-      | 36000000   | The amount of levy you need to pay for this financial year is £36,000.  |
-      | 999999999  | The amount of levy you need to pay for this financial year is £36,000.  |
-      | 1000000000 | The amount of levy you need to pay for this financial year is £250,000. |
+      | UK Revenue | Expected content                                           |
+      | 10200000   | You need to pay £10,000 for the 2022-2023 financial year.  |
```

Comment:
2022-2023 is dynamic based on the financial year of the obligation that the return is for, so does it make sense to include it like this (I don't know if you can make it dynamic)?

Reply:
Discussed via Slack